From 28e95089c3d2456425033f20c5cc598cdd7ee058 Mon Sep 17 00:00:00 2001
From: Hariharan K <quic_harihk@quicinc.com>
Date: Tue, 19 Dec 2023 20:59:28 +0530
Subject: [PATCH] arm64: dts: Update build support for rdp455

Updating the Makefile to build dtb for
rdp455-c11 and rdp455-c12 dts files.

Change-Id: I81552316f21243f08c2a5664c87ef79c57090cd0
Signed-off-by: Hariharan K <quic_harihk@quicinc.com>
---
 arch/arm/boot/dts/Makefile        | 3 ++-
 arch/arm64/boot/dts/qcom/Makefile | 3 ++-
 2 files changed, 4 insertions(+), 2 deletions(-)

diff --git a/arch/arm/boot/dts/Makefile b/arch/arm/boot/dts/Makefile
index fd5c83c23c5d..8b14e36f8f24 100644
--- a/arch/arm/boot/dts/Makefile
+++ b/arch/arm/boot/dts/Makefile
@@ -1057,7 +1057,8 @@ dtb-$(CONFIG_ARCH_QCOM) += \
 	ipq9574-rdp449.dtb \
 	ipq9574-rdp453.dtb \
 	ipq9574-rdp454.dtb \
-	ipq9574-rdp455.dtb \
+	ipq9574-rdp455-c11.dtb \
+	ipq9574-rdp455-c12.dtb \
 	ipq9574-rdp456.dtb \
 	ipq9574-rdp457.dtb \
 	ipq9574-rdp458.dtb \
diff --git a/arch/arm64/boot/dts/qcom/Makefile b/arch/arm64/boot/dts/qcom/Makefile
index e7b90f8a4195..063931f1a5c6 100644
--- a/arch/arm64/boot/dts/qcom/Makefile
+++ b/arch/arm64/boot/dts/qcom/Makefile
@@ -32,7 +32,8 @@ dtb-$(CONFIG_ARCH_QCOM)	+= ipq9574-rdp437.dtb
 dtb-$(CONFIG_ARCH_QCOM)	+= ipq9574-rdp449.dtb
 dtb-$(CONFIG_ARCH_QCOM)	+= ipq9574-rdp453.dtb
 dtb-$(CONFIG_ARCH_QCOM)	+= ipq9574-rdp454.dtb
-dtb-$(CONFIG_ARCH_QCOM)	+= ipq9574-rdp455.dtb
+dtb-$(CONFIG_ARCH_QCOM)	+= ipq9574-rdp455-c11.dtb
+dtb-$(CONFIG_ARCH_QCOM)	+= ipq9574-rdp455-c12.dtb
 dtb-$(CONFIG_ARCH_QCOM)	+= ipq9574-rdp456.dtb
 dtb-$(CONFIG_ARCH_QCOM)	+= ipq9574-rdp457.dtb
 dtb-$(CONFIG_ARCH_QCOM)	+= ipq9574-rdp458.dtb
-- 
2.34.1

